core/__init__.py

```diff
@@ -171,8 +170,11 @@ def start():
 
     enter_normal_mode()
     logger.debug("Entering main loop")
+
+    # Monitor Power Level
+    power_monitoring_thread = ThreadHandler(target=partial(power_watchdog,
```

Comment:
you are passing too many arguments to `partial` it should be 

`ThreadHandler(target=partial(power_watchdog), name='monitoring_power', parent_logger=logger)`

Reply:
Sorry for prematurely approving your changes, I missed this one change

Reply:
Thanks, I fixed it